osu.Game/Overlays/ModSelection/MultimodButton.cs

```diff
@@ -0,0 +1,131 @@
+﻿//Copyright (c) 2007-2016 ppy Pty Ltd <contact@ppy.sh>.
+//Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System.Collections.Generic;
+using OpenTK.Graphics;
+using osu.Framework;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Primitives;
+using osu.Framework.Input;
+using osu.Game.Modes;
+using osu.Game.Modes.UI;
+
+namespace osu.Game.Overlays.ModSelection
+{
+    class MultimodButton : ClickableContainer, IStateful<ModButtonState>

```

Comment:
I'd probably combine this with ModButton (and even if i was leaving it as a separate class i would a) call it MultiModButton and b) make it derive ModButton).